scripts/publish_pr.py

```
@@ -0,0 +1,389 @@
+    branch: str,
+    default_branch: str,
+    title: str,
+    body: str,
+) -> Optional[str]:
+    """Create a PR and return the URL."""
+    try:
+        # Write body to temp file to avoid shell quoting issues
+        body_file = Path.home() / f".pr-body-{os.getpid()}.txt"
+        body_file.write_text(body, encoding="utf-8")
+
+        result = subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)